docs/software/unsupported.md

```diff
@@ -0,0 +1,20 @@
+[](){ref-software-unsupported}
+# Unsupported Software
+
+This page documents frequently-requested applications that CSCS won't provide pre-built on Alps as [supported software][ref-support-apps].
+
+!!! warning "license terms"
+    Often it is impossible or difficult to provide software for licensing reasons.
+
+    Note that if users are asked to install licensed software because CSCS can't, it is the responsibility of users to ensure that the software is only available to them personally, or to members of their group who are also permitted by the license to access the software.
```

Comment:
Do we already have a section on permissions and ACL as in the ["old" documentation](https://confluence.cscs.ch/spaces/KB/pages/850200817/How+to+give+other+groups+or+users+access+to+your+data+on+CSCS+systems)? It would be worth linking it here.